src/mercury_engine_data_structures/formats/bmsld.py

```diff
@@ -36,36 +37,37 @@
 
 Components = {
     "TRIGGER": Struct(
-        command=StrId,
-        arguments=make_vector(FunctionArgument),
+        "command" / StrId,
+        "arguments" / make_vector(FunctionArgument),
     ),
     "SPAWNGROUP": Struct(
-        command=StrId,
-        arguments=make_vector(FunctionArgument),
+        "command" / StrId,
+        "arguments" / make_vector(FunctionArgument),
     ),
     "SPAWNPOINT": Struct(
-        command=StrId,
-        arguments=make_vector(FunctionArgument),
+        "command" / StrId,
+        "arguments" / make_vector(FunctionArgument),
     ),
     "STARTPOINT": Struct(
-        command=StrId,
-        arguments=make_vector(FunctionArgument),
+        "command" / StrId,
+        "arguments" / make_vector(FunctionArgument),
     ),
     "MODELUPDATER": Struct(
-        command=StrId,
-        arguments=make_vector(FunctionArgument),
+        "command" / StrId,
+        "arguments" / make_vector(FunctionArgument),
     ),
 }
 
 ProperActor = Struct(
-    type=StrId,
-    position=CVector3D,
-    rotation=CVector3D,
-    components=make_vector(
+    "type" / StrId,
+    "position" / CVector3D,
+    "rotation" / CVector3D,
+    "components"
+    / make_vector(
         Struct(
-            component_type=StrId,
-            command=StrId,
-            arguments=make_vector(FunctionArgument),
+            "component_type" / StrId,
+            "command" / StrId,
+            "arguments" / make_vector(FunctionArgument),
```

Comment:
I really don't love how this is set up but I'm gonna need some time to think about a better way to do it. it's hard to access a specific function call and all the typing has to be kept track of manually. hm